<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScriptOptions</class>
 <widget class="QWidget" name="ScriptOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="runningScriptsLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>287</width>
         <height>176</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Helvetica</family>
        </font>
       </property>
       <property name="text">
        <string>Running Scripts</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="scriptOptionsClose">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../resources/resources.qrc">:/images/close.svg</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QVBoxLayout" name="currentlyRunning">
       <item>
        <widget class="QLabel" name="currentlyRunningLabel">
         <property name="text">
          <string>Currently running</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="scriptControls">
         <property name="standardButtons">
          <set>QDialogButtonBox::NoButton</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="recentScripts">
     <item>
      <widget class="QLabel" name="recentScriptsLabel">
       <property name="text">
        <string>Recently loaded</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
